pdns/dnsmessage.proto

```diff
@@ -126,6 +126,36 @@ message PBDNSMessage {
     required MetaValue value = 2;
   }
   repeated Meta meta = 22;                      // Arbitrary meta-data - to be used in future rather than adding new fields all the time
+
+  // The well known EventTrace event numbers
+  enum EventType {
+    CustomEvent = 0;                            // A custom event
+    RecRecv = 1;                                // A request was received by the recursor process
```

Comment:
We might want to rename that event so that dnsdist and the auth can later use the same event? Something like `RequestReceived`?

Reply:
> I like that new feature very much! Did you get a chance to measure the overhead introduced by this tracing, by any chance?

not yet. My plan wast to get the semantics and code in shape and then do measurements.

Reply:
> We might want to rename that event so that dnsdist and the auth can later use the same event? Something like `RequestReceived`?

Yes, makes sense